inlong-sort/sort-connectors/doris/src/main/java/org/apache/inlong/sort/doris/backend/BackendClient.java

```diff
@@ -0,0 +1,225 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.inlong.sort.doris.backend;
+
+import org.apache.inlong.sort.doris.util.ErrorMessages;
+import org.apache.doris.thrift.TDorisExternalService;
+import org.apache.doris.thrift.TScanBatchResult;
+import org.apache.doris.thrift.TScanCloseParams;
+import org.apache.doris.thrift.TScanCloseResult;
+import org.apache.doris.thrift.TScanNextBatchParams;
+import org.apache.doris.thrift.TScanOpenParams;
+import org.apache.doris.thrift.TScanOpenResult;
+import org.apache.doris.thrift.TStatusCode;
+import org.apache.inlong.sort.doris.cfg.ConfigurationOptions;
+import org.apache.inlong.sort.doris.cfg.DorisReadOptions;
+import org.apache.inlong.sort.doris.exception.ConnectedFailedException;
+import org.apache.inlong.sort.doris.exception.DorisException;
+import org.apache.inlong.sort.doris.exception.DorisInternalException;
+import org.apache.inlong.sort.doris.serialization.Routing;
+import org.apache.thrift.TException;
+import org.apache.thrift.protocol.TBinaryProtocol;
+import org.apache.thrift.protocol.TProtocol;
+import org.apache.thrift.transport.TSocket;
+import org.apache.thrift.transport.TTransport;
+import org.apache.thrift.transport.TTransportException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Client to request Doris BE
+ */
+public class BackendClient {
```

Comment:
Can you find a public jar to replace all the classes?

Reply:
> Can you find a public jar to replace all the classes?

I can compile a doris-flink-connector jar compatible with scala2.11 version, do we have a public maven repository for uploading(doris official community has not released scala2.11 version for flink1.13 version)